actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -282,8 +285,11 @@ def image_alt(src)
       #   video_tag(["trailer.ogg", "trailer.flv"], size: "160x120")
       #   # => <video height="120" width="160"><source src="/videos/trailer.ogg" /><source src="/videos/trailer.flv" /></video>
       def video_tag(*sources)
```

Comment:
We could remove the options wrangling with a keyword argument:

```ruby
def video_tag(*sources, skip_pipeline_on_poster: nil)
  multiple_sources_tag_builder("video", sources) do |options|
    options[:poster] = path_to_image(options[:poster], skip_pipeline: skip_pipeline_on_poster) if options[:poster]
    options[:width], options[:height] = extract_dimensions(options.delete(:size)) if options[:size]
  end
end
```

I think that would also save us a block argument shadowing local variable warning from `options`. 😁